src/web100srv.c

```diff
@@ -1393,72 +1302,18 @@ void child_process(int parent_pipe, SSL_CTX *ssl_context, int ctlsockfd) {
     exit(1);
   }
 #endif
-  for (;;) {
-    // the read() could return if an interrupt was caught.  This
-    // condition should be checked for and the read() restarted if
-    // necessary
-    retcode = read(parent_pipe, &parent_message, sizeof(int));
-    log_println(6, "Child %d got '%d' from parent", getpid(), parent_message);
-    if ((retcode == -1) && (errno == EINTR)) continue;
-    if (parent_message == SRV_QUEUE_TEST_STARTS_NOW) {
-      send_json_message_any(&ctl, SRV_QUEUE, SRV_QUEUE_TEST_STARTS_NOW_STR,
+  // Wait in the queue until the child process is told to start
+  while ((parent_message = read_from_parent(parent_pipe)) !=
+         SRV_QUEUE_TEST_STARTS_NOW) {
```

Comment:
>>Also this might not have come across but I meant to be calling for deleting the unused #defines

>They aren't all dead and gone.
>They are gone from my code, but other code uses them.

```
grep SRV_QUEUE.*STR ./* -R
./src/protocol.h:#define SRV_QUEUE_TEST_STARTS_NOW_STR  "0"
./src/protocol.h:#define SRV_QUEUE_SERVER_FAULT_STR "9977"
./src/protocol.h:#define SRV_QUEUE_SERVER_BUSY_STR "9988"
./src/protocol.h:#define SRV_QUEUE_HEARTBEAT_STR "9990"
./src/protocol.h:#define SRV_QUEUE_SERVER_BUSY_60s_STR "9999"
./src/web100srv.c:  if (send_json_message_any(conn, SRV_QUEUE, SRV_QUEUE_HEARTBEAT_STR,
./src/web100srv.c:                            strlen(SRV_QUEUE_HEARTBEAT_STR),
./src/web100srv.c:  if (send_json_message_any(&ctl, SRV_QUEUE, SRV_QUEUE_TEST_STARTS_NOW_STR,
./src/web100srv.c:                            strlen(SRV_QUEUE_TEST_STARTS_NOW_STR),
```

`SRV_QUEUE_SERVER_FAULT_STR`, `SRV_QUEUE_SERVER_BUSY_STR`, `SRV_QUEUE_SERVER_BUSY_60s_STR` are gone (I believe) as a result of this pull request